default.xml

```diff
@@ -4,16 +4,20 @@
     <remote name="prpl" fetch="https://github.com/prplfoundation/" pushurl="ssh://git@github.com/prplfoundation" />
     <remote name="nanomsg" fetch="https://github.com/nanomsg/" />
     <remote name="rurban" fetch="https://github.com/rurban/" />
+    <remote name="boardfarm" fetch="https://github.com/mattsm/" />
 
-	<default revision="master" remote="prpl" sync-j="4" />
+    <default revision="master" remote="prpl" sync-j="4" />
 
-	<project path="prplMesh" name="prplMesh"/>
-	<project path="hostap" name="hostap" revision="prpl-mesh"/>
-	<project path="hostap/dwpal" name="dwpal"/>
+    <project path="prplMesh" name="prplMesh"/>
+    <project path="hostap" name="hostap" revision="prpl-mesh"/>
+    <project path="hostap/dwpal" name="dwpal"/>
 
-	<!-- upstream nng -->
-	<project path="3rdparty/nng" remote="nanomsg" name="nng" revision="90467583b7544b68483334070518e29b00ec6d81" />
+    <!-- upstream nng -->
+    <project path="3rdparty/nng" remote="nanomsg" name="nng" revision="90467583b7544b68483334070518e29b00ec6d81" />
 
-	<!-- upstream safelibc -->
-	<project path="3rdparty/safeclib" remote="rurban"  name="safeclib" revision="ab130d7376267b6deb55e194746fc08d045afa61" />
+    <!-- upstream safelibc -->
+    <project path="3rdparty/safeclib" remote="rurban"  name="safeclib" revision="ab130d7376267b6deb55e194746fc08d045afa61" />
+
+    <!-- upstream boardfarm -->
+    <project path="3rdparty/boardfarm" remote="boardfarm" name="boardfarm" revision="228c3434fe12d596d740a6ab6635dd5f44fa2ae7" />
```

Comment:
(nitpick) I don't think it's useful to put it under 3rdparty.

Reply:
> (nitpick) I don't think it's useful to put it under 3rdparty.

I'll move boardfarm dir up one level(repo root dir), perhaps that will be more usefull.
Do you agree?